oas-generator/oas-generator-spring/src/test/java/org/apache/servicecomb/toolkit/generator/SpringAnnotationProcessorTest.java

```diff
@@ -55,13 +57,15 @@
 
   @Test
   public void getHttpMethod() throws NoSuchMethodException {
```

Comment:
This test is too big. Write individual method for each `@RequestMapping` property and a test for properties inheritance, eg, Class level `@RequestMapping` + Method level `RequestMapping`, will be clearer